roles/datadog-builder/tasks/datadog-builder.yml

```diff
@@ -20,12 +20,26 @@
     - /opt/venvs
     - /etc/datadog-builder
 
+- name: create datadog-builder group
+  group:
+    name: datadog-builder
+    state: present
+
 - name: create datadog-builder user
   user:
     name: datadog-builder
+    group: datadog-builder
     home: /var/lib/datadog-builder
     state: present
 
+- name: Create log file
+  file:
+    path: /var/log/datadog-builder
+    state: directory
+    mode: 0755
```

Comment:
Why create the group if the mode is 755? I would expect this to be either 775 or 750 if we are also creating the group.